### PR TITLE
Adding the correct version of wgrib2 on Orion to convert GRIB2 to netcdf

### PR DESCRIPTION
**Pull request by mrinalbiswas**
## Description of changes
The RT on Orion was failing on the ww3 tasks because it was unable to convert the GRIB2 gfs files to NETCDF. 

Error: 
03/17 09:42:50.410 hafs.ww3init (run.py:349) INFO: Starting: batchexe('/apps/contrib/NCEP/libs/hpc-stack/intel-2018.4/impi-2018.4/wgrib2/2.0.8/bin/wgrib2')['/work/noaa/hwrf/noscrub/hafs-input/COMGFSv16/gfs.20200825/12/atmos/gfs.t12z.pgrb2.0p25.f000','-i','-append','-netcdf','[gfs.uvgrd10m.nc](http://gfs.uvgrd10m.nc/)'].in('585:363177507:d=2020082512:UGRD:10 m ...'...,string=True)
netcdf package not installed

The PR will fix the error. 

03/17 09:42:50.434 hafs.ww3init (run.py:453) INFO:   - exit status

## Contributors (optional)
If others worked on this PR besides the author, please include their user names here (using @Mention if possible).

## Tests conducted
What testing has been conducted on the PR thus far? Describe the nature of any scientific or technical tests, including relevant details about the configuration(s) (e.g., cold versus warm start, number of cycles, forecast length, whether data assimilation was performed, etc). What platform(s) were used for testing?
Tested on Orion

## Application-level regression test status
Running the HAFS application-level regression tests is currently performed by code reviewers after the developer creates the initial PR. As regression tests are conducted, the testers should use the checklist below to indicate **successful** regression tests. You may add other tests as needed. If a test fails, do not check the box. Instead, describe the failure in the PR comments, noting the platform where the test failed.

- [ ] Jet
- [ ] Hera
- [x] Orion PASSED
- [ ] WCOSS2

### CODE REVIEW

**Review by KathrynNewman — APPROVED** [review]

File: jobs/JHAFS_WAV_PREP [hunk 1]
```diff
@@ -33,6 +33,11 @@ if [ "${RUN_ENVIR^^}" != "NCO" ]; then
   source ${USHhafs}/hafs_pre_job.sh.inc
   export machine=${WHERE_AM_I:-wcoss2}
   module list
+  # A temporary workaround due to the hpc-stack version of wgrib2 does not work
+  # on Orion to convert grib2 files into netcdf format.
+  if [ ${machine} = orion ]; then
+    export WGRIB2=/apps/wgrib-2.0.8/wgrib2/wgrib2
```
Comment (BinLiu-NOAA) [inline jobs/JHAFS_WAV_PREP]: @mrinalbiswas and @BijuThomas-NOAA, will this /apps/wgrib-2.0.8/wgrib2/wgrib2 version works for all other HAFS jobs on Orion, if so, we may change the wgrib2 version in the modulefiles/hafs.orion.lua instead.

Comment (mrinalbiswas) [reply jobs/JHAFS_WAV_PREP]: @BinLiu-NOAA I can run with this version and let you know. The path on hafs.orion.lua directs wgrib2 to setenv("WGRIB2", "/apps/contrib/NCEP/libs/hpc-stack/intel-2018.4/impi-2018.4/wgrib2/2.0.8/bin/wgrib2"). I am not sure whether this is a working version of wgrib2 or not. The default on Orion points to: /apps/wgrib2-3.0.2/wgrib2/wgrib2.

**Review by panll — APPROVED** [review]

File: jobs/JHAFS_WAV_PREP (hunk 1 above)
Comment (BinLiu-NOAA) [reply jobs/JHAFS_WAV_PREP]: @mrinalbiswas, appreciate it if you could help to test if the /apps/wgrib2-3.0.2/wgrib2/wgrib2 version and its corresponding module version can work for the whole HAFS workflow on Orion. In such case, we can just implement this fix inside hafs.orion.lua. Thanks!

Comment (KathrynNewman) [reply jobs/JHAFS_WAV_PREP]: @mrinalbiswas - I originally approved this PR before Bin's suggestion. I requested a re-review once this has been tested. Thanks!

**Comment by mrinalbiswas** [comment]
@BinLiu-NOAA I ran the RT successfully with only the wgrib2 changes in modulefiles/hafs.orion.lua. So we do not need the changes in the jobs/JHAFS_WAVE_PREP.

**Comment by mrinalbiswas** [comment]
@BinLiu-NOAA @ZhanZhang-NOAA @KathrynNewman @linlin-pan Please review the PR. RTs passed successfully on Orion

**Review by BinLiu-NOAA — APPROVED** [review]
Looks good to me. Thanks for testing this out and fixing it, @mrinalbiswas!

**Comment by BinLiu-NOAA** [comment]
@mrinalbiswas, Could you please help update your PR branch with the latest production/hafs.v1 branch? Meanwhile, if would be beneficial if you can also update the PR description properly. After that, I we can "squash and merge" this PR. Thanks!

**Review by panll — APPROVED** [review]

**Review by KathrynNewman — APPROVED** [review]

**Comment by mrinalbiswas** [comment]
@BinLiu-NOAA Updated my PR branch and updated the PR description. Ready for review.

**Pull request merged**